sys-kernel/genkernel/genkernel-9999.ebuild

```diff
@@ -128,11 +129,9 @@ RDEPEND="${PYTHON_DEPS}
 	dev-build/libtool
 	virtual/pkgconfig
 	elibc_glibc? ( sys-libs/glibc[static-libs(+)] )
-	firmware? ( sys-kernel/linux-firmware )"
+	firmware? ( sys-kernel/linux-firmware )
+"
 
-if [[ ${PV} == 9999* ]]; then
-	DEPEND="${DEPEND} app-text/asciidoc"
-fi
```

Comment:
no, this is still required in DEPEND

Reply:
I've moved it up to the main DEPEND definition [here](https://github.com/gentoo/gentoo/pull/35089/files/1127ffdeea9c6b2120bddbbc1b6e428e0290e94f#diff-9cb3123499b5ef6595e1a2fd0b48c1b758f7789f1f5e46340db76b18baaa4eccR120).

By the way, shouldn't we have this as BDEPEND instead, as it isn't required for runtime?

Reply:
yes, BDEPEND and/or DEPEND, but certainly not in RDEPEND where you've put it.

Reply:
I see what you mean now! I'll change it accordingly